Added support for uuid

SergPonomar opened the pull request:
Added generic return type in 'populate' funcs for support primary keys other than u64 (uuid).

kenkoooo commented:
Thank you for your pull request. Could you please explain why the return type needed to be ToString?

SergPonomar commented:
Hi, kenkoo.
I'm using Uuid as primary key type in my db. In my project I use SeaOrm, which return actual value of primary key (last_insert_id) https://docs.rs/sea-orm/0.12.6/sea_orm/struct.InsertResult.html. I cannot cast uuid to u64, because he represent 128bit value https://docs.rs/uuid/latest/uuid. It's be useful If you add support to this type.
I'm suggest to add generic type with ToString because it's simple solution for support both u64 and Uuid. I see that ids anyway convert to String before inserted in name_resolver.
I runed test in your repo, and also tried use this solution in my project, works fine.

kenkoooo commented:
I understand completely. Your contribution makes perfect sense to me. Thank you.

SergPonomar commented:
I fixed the code formatting by running rustfmt.
Also removed redundant code suggested by clippy https://rust-lang.github.io/rust-clippy/master/index.html#/useless_conversion. The zip and from_iter functions accept any arguments that implements IntoIterator, there is no need to call "into_iter".
After this changes, the ci action should become "happy".

kenkoooo commented:
We are preparing to publish this feature to crates.io. Thank you for your contribution!
